src/coreclr/inc/corinfo.h

```diff
@@ -317,8 +317,8 @@ struct SYSTEMV_AMD64_CORINFO_STRUCT_REG_PASSING_DESCRIPTOR
     }
 };
 
-// StructFloadFieldInfoFlags: used on LoongArch64 architecture by `getLoongArch64PassStructInRegisterFlags` and
-// `getRISCV64PassStructInRegisterFlags` API to convey struct argument passing information.
+// StructFloatFieldInfoFlags: used on LoongArch64 and RISC-V architecture as a legacy representation of
```

Comment:
Could you please move the definitions that are not part of the JIT/EE interface out of this file? They should live somewhere under VM. corinfo.h should be only used for definitions that are part of JIT/EE interface